lib/win32/taskscheduler.rb

```diff
@@ -10,250 +11,106 @@ module Win32
   # The TaskScheduler class encapsulates a Windows scheduled task
   class TaskScheduler
     include Windows::TaskSchedulerHelper
-
-    # The version of the win32-taskscheduler library
-    VERSION = '0.3.2'.freeze
```

Comment:
@Nimesh-Msys I'm going to merge this, but consider moving this constant back. This is specific to the library and let's you ask `Win32::TaskScheduler::VERSION`. I think moving it is an unnecessary breaking change.

Reply:
@Nimesh-Msys we're going to want to bump this to 1.0.0 with all the changes too.